app/themes/dashboard/ui/styles/theme.less

```diff
@@ -38,4 +38,16 @@ z-index: 10;
 
 .main-header #logo {
    text-indent: 99999px;
-}
\ No newline at end of file
+}
+
+.pre-xml {
+  background-color: #ecf0f5;
+}
+
+.nav-pills>li>a {
+  border-radius: 4px !important;
```

Comment:
The design is for square buttons and not rounded ones. This should be removed. Thanks!